src/orange/BoundingBoxIO.json.hh

```diff
@@ -7,29 +7,62 @@
 //---------------------------------------------------------------------------//
 #pragma once
 
+#include <limits>
 #include <nlohmann/json.hpp>
 
 #include "corecel/cont/ArrayIO.json.hh"
 
 #include "BoundingBox.hh"
 
+namespace
+{
```

Comment:
It's (always?) a good idea to put anonymous namespaces *inside* of the main namespace you're using: that way it can use the same scope as a regular function, but it simply won't generate a symbol.

Reply:
How about I fix it in the next MR so we don't have to rerun CI?

Reply:
It's fixed now. Just an informational message.